From: John Crispin <john@phrozen.org>
Subject: [PATCH] ath79: add new OF only target for QCA MIPS silicon

This target aims to replace ar71xx mid-term. The big part that is still
missing is making the MMIO/AHB wifi work using OF. NAND and mikrotik
subtargets will follow.

Submitted-by: John Crispin <john@phrozen.org>
---
 drivers/net/ethernet/atheros/Kconfig          | 8 +-------
 drivers/net/ethernet/atheros/Makefile         | 2 +-
 2 files changed, 2 insertions(+), 8 deletions(-)

--- a/drivers/net/ethernet/atheros/Kconfig
+++ b/drivers/net/ethernet/atheros/Kconfig
@@ -26,6 +26,8 @@ config AG71XX
 	  If you wish to compile a kernel for AR7XXX/91XXX and enable
 	  ethernet support, then you should always answer Y to this.
 
+source "drivers/net/ethernet/atheros/ag71xx/Kconfig"
+
 config ATL2
 	tristate "Atheros L2 Fast Ethernet support"
 	depends on PCI
--- a/drivers/net/ethernet/atheros/Makefile
+++ b/drivers/net/ethernet/atheros/Makefile
@@ -4,6 +4,7 @@
 #
 
 obj-$(CONFIG_AG71XX) += ag71xx.o
+obj-$(CONFIG_AG71XX_LEGACY) += ag71xx/
 obj-$(CONFIG_ATL1) += atlx/
 obj-$(CONFIG_ATL2) += atlx/
 obj-$(CONFIG_ATL1E) += atl1e/
